src/TextInputWidget.cc

```diff
@@ -165,9 +162,7 @@ FilteredTextEdit::insertFromMimeData(const QMimeData *source)
                         qWarning()
                           << "Clipboard does not contain any valid file paths:" << source->urls();
                 }
-        } else if (source->hasFormat("x-special/gnome-copied-files") &&
-                   QFileInfo{source->text()}.exists()) {
-                // Generic file for any platform.
+        } else if (source->hasFormat("x-special/gnome-copied-files")) {
                 // Special case for X11 users. See "Notes for X11 Users" in source.
                 // Source: http://doc.qt.io/qt-5/qclipboard.html
                 previewDialog_.setPreview(source->text());
```

Comment:
Again: why use the `text/plain` data instead of the `x-special/gnome-copied-files` data?